src/Oracles/Path.hs

```diff
@@ -28,7 +28,7 @@ systemBuilderPath builder = case builder of
     Cc  _  Stage0   -> fromKey "system-cc"
     Cc  _  _        -> fromKey "cc"
     -- We can't ask configure for the path to configure!
-    Configure _     -> return "bash configure"
+    Configure _     -> return "sh configure"
```

Comment:
Nope, as I mention in the commit message I don't think there's any reason why we need to use `bash` at all here. `sh` should be perfectly sufficient unless I've missed something.